src/package_io/parse_input.star

```diff
@@ -32,8 +32,10 @@ def parse_input(plan, input_args):
 	if result.get("mev_type") in ("mock", "full"):
 		result = enrich_mev_extra_params(result, MEV_BOOST_SERVICE_NAME_PREFIX, FLASHBOTS_MEV_BOOST_PORT, result.get("mev_type"))
 
-	if result.get("mev_type") == "full" and result["network_params"]["capella_fork_epoch"] == 0:
-		fail("capella_fork_epoch needs to be set to a non-zero value when using full MEV, set it using network_params.capella_fork_epoch")
+	#if result.get("mev_type") == "full" and result["network_params"]["capella_fork_epoch"] == 0:
```

Comment:
why has this been commented?

Reply:
Maybe this can just be removed? I think flashbots added support for MEV with Capella at 0

Reply:
Ahhhh good catch! Commented jsut for testing